### ISSUE TITLE
preview on youtube 

### ISSUE THREAD

**Issue by furuikisui** [extesy/hoverzoom#268]
The preview function on youtube.com does not work.
When you place the pointer over a video, the image shown is blank.
Previously if it worked, but not now.

## Steps to reproduce the problem (include image url here)
1.
2.
3.

**Comment by Sacke** [comment]
For me, the extension very seldom shows an enlargement. It starts with the "working" animation and ends in nothing.

**Comment by DnagAnthony** [comment]
Yes this has been present since last month as far as i can remember. a Fix would be nice.
